micro_sam/sam_annotator/annotator_tracking.py

```diff
@@ -101,7 +101,7 @@ class AnnotatorTracking(_AnnotatorBase):
     # The tracking annotator needs different settings for the prompt layers
     # to support the additional tracking state.
     # That's why we over-ride this function.
-    def _create_layers(self):
+    def _require_layers(self):
```

Comment:
We need to add the logic to prevent adding layers multiple times here as well. And you should have the same function signature as in the other case.